src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DoNotPassLiteralsAsLocalizedParameters.cs

```diff
@@ -66,7 +66,7 @@ public override void Initialize(AnalysisContext context)
                         return;
                     }
 
-                    var lazyValueContentResult = new Lazy<DataFlowAnalysisResult<ValueContentBlockAnalysisResult, ValueContentAbstractValue>?>(
+                    Lazy<DataFlowAnalysisResult<ValueContentBlockAnalysisResult, ValueContentAbstractValue>?> lazyValueContentResult = new Lazy<DataFlowAnalysisResult<ValueContentBlockAnalysisResult, ValueContentAbstractValue>?>(
```

Comment:
Same here - this seems unfortunate.

Reply:
➡️ Covered by https://github.com/dotnet/roslyn-analyzers/pull/3400#discussion_r394759842. The compiler requires this change.